seed-element.html

```diff
@@ -1,97 +1,110 @@
 <!--
-    @license
-    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
-    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-    Code distributed by Google as part of the polymer project is also
-    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 <link rel="import" href="../polymer/polymer.html">
 
 <!--
-Element providing solution to no problem in particular.
+An element providing a solution to no problem in particular.
 
-##### Example
+Example:
 
     <seed-element></seed-element>
 
-@element seed-element
-@blurb Element providing solution to no problem in particular.
-@status alpha
-@homepage http://polymerlabs.github.io/seed-element
 -->
-<polymer-element name="seed-element" attributes="notitle author">
+<dom-module id="seed-element">
 
-  <template>
-
-    <link rel="stylesheet" href="seed-element.css">
+  <style>
+    seed-element {
+      display: block;
+      box-sizing: border-box;
```

Comment:
:host works in 0.8?

On Sat, Feb 21, 2015, 11:26 Addy Osmani <notifications@github.com> wrote:

> In seed-element.html
> <https://github.com/PolymerLabs/seed-element/pull/41#discussion_r25124887>
> :
>
> >
> > -  <template>
> > -
> > -    <link rel="stylesheet" href="seed-element.css">
> > +  <style>
> > +    seed-element {
> > +      display: block;
> > +      box-sizing: border-box;
>
>  set box-sizing: border-box; targeting :host or the element's tag name.
>
> [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PolymerLabs/seed-element/pull/41/files#r25124887>.
>